cedar-14/bin/cedar-14.sh

```diff
@@ -191,6 +191,7 @@ apt-get install -y --force-yes \
     iputils-tracepath \
     language-pack-en \
     libbz2-dev \
+    libc-client2007e \
```

Comment:
Should `libc-client2007e-dev` also be here too, given that Cedar-14's image is combined runtime+buildtime image, and `libc-client2007e-dev` has been added to the build time variants below?

Reply:
We've generally not done that until really necessary in order to keep the image size reasonable.